apache/couchdb-fauxton
# Add option to create target db as partitioned in replication page

## Overview

In the Create Replication form, shows a 'Partitioned' checkbox when the target db is a new local/remote database. The checkbox is not displayed when selecting an existing local/remote target db.
When the checkbox is selected, the new `_replicator` doc is created with the field `"create_target_params": { "partitioned": true }`.

If the local server does not support partitioned dbs, the checkbox is still displayed but it is disabled and a tooltip explains that the feature is not enabled. 
The same check is not done for remote dbs, so the checkbox is always enabled in this case.

## Testing recommendations

test 1
- Create a new replication
- Select existing local target db, and verify Partitioned option is not displayed
- Select existing remote target db, and verify Partitioned option is not displayed
- Select new remote target db, and verify Partitioned option is displayed and enabled
- Select new local target db, and verify Partitioned option is displayed
  - The checkbox should be disabled if the partitioned db feature is not available on the local server

test 2
- Create a new replication
- Select new remote target db and select the Partitioned option
- Verify the created doc contains `"create_target_params": { "partitioned": true }` 


## Checklist
